blackjax/inference/hmc/hmc.py

```diff
@@ -79,21 +92,46 @@ def kernel(
 
     """
 
-    def potential_fn(x):
-        return -logprob_fn(x)
+    def kernel(
```

Comment:
I think we should rename kernel here to something else (`kernel_fn` to indicate it is a PyTree -> PyTree function?) or the top level function to `gen_kernel`

Reply:
Renamed it to `one_step`. Will do for now.